ui/server/src/main/scala/pl/touk/nussknacker/ui/validation/DeploymentCommentValidator.scala

```diff
@@ -0,0 +1,55 @@
+package pl.touk.nussknacker.ui.validation
```

Comment:
It is not a good package. This `validation` means scenario validation. Move it please to  `pl.touk.nussknacker.ui.process.deployment`

Reply:
moved it, thanks